src/app/static/src/js/components/workflowWizard/step.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <div v-if="active && hasValidComponent">
+        <div>
+            <slot></slot>
+        </div>
+        <div class="pt-4">
+            <button id="cancel-workflow" type="button" class="btn btn-sm btn-secondary"
+                    @click="cancel">Cancel
+            </button>
+            <button id="previous-workflow" v-if="hasVisibility.back" type="button"
+                    class="btn btn-sm btn-primary"
+                    @click="back">Back
+            </button>
+            <button id="next-workflow" type="button" class="btn btn-sm btn-success"
+                    :class="{disabled: !valid}"
+                    @click="next"> {{ hasVisibility.next ? "Next" : "Run workflow" }}
```

Comment:
As commented previously, the label for the final "next" button should be configurable when creating the wizard - removing all knowledge of workflows from this component.

Reply:
done()